--- zfs-kmod.spec.in	2023-04-20 02:31:29.000000000 +0300
+++ zfs-kmod.spec.in_	2023-05-05 16:45:11.577793050 +0300
@@ -76,12 +76,6 @@
 
 %else
 
-# Building local packages attempt to to use the installed kernel.
-%{?rhel:BuildRequires: kernel-devel}
-%{?fedora:BuildRequires: kernel-devel}
-%{?openEuler:BuildRequires: kernel-devel}
-%{?suse_version:BuildRequires: kernel-source}
-
 %if !%{defined kernels} && !%{defined build_src_rpm}
     %if 0%{?rhel}%{?fedora}%{?suse_version}%{?openEuler}
         %define kernels %(ls -1 /usr/src/kernels)
